alvr/graphics/src/lib.rs

```diff
@@ -286,15 +286,13 @@ impl GraphicsContext {
                 let gl_context = gl::Context::from_loader_function(|fn_name| {
                     egl_instance
                         .get_proc_address(fn_name)
-                        .map(|f| f as *const c_void)
-                        .unwrap_or(ptr::null())
+                        .map_or(ptr::null(), |f| f as *const c_void)
```

Comment:
Could this maybe use `.cast()`?

Reply:
No, `.cast()` casts from `*... T` to `*... U`, this casts from `fn()`, otherwise the lint would've caught it too ;)